<launch>
    <arg name="open_rviz" default="true"/>
    <include file="$(find turtlebot3_gazebo)/launch/multi_turtlebot3_nav.launch">        
        <arg name="ns" value="tb3_0"/>
    </include> 
    <include file="$(find turtlebot3_gazebo)/launch/multi_turtlebot3_nav.launch">        
        <arg name="ns" value="tb3_1"/>
    </include> 
    <include file="$(find turtlebot3_gazebo)/launch/multi_turtlebot3_nav.launch">        
        <arg name="ns" value="tb3_2"/>
    </include> 

    
<!-- rviz -->
  <group if="$(arg open_rviz)"> 
    <node pkg="rviz" type="rviz" name="rviz" required="true"
          args="-d $(find turtlebot3_navigation)/rviz/turtlebot3_navigation.rviz"/>
  </group>
</launch>